digitalpublicgoods/e-heza.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "E-Heza",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://www.apache.org/licenses/LICENSE-2.0"
```

Comment:
This is a direct link to Apache License. We would need a link to their own license or other evidence of ownership.
I see copyright is actually held by "TIP Global Health".